ChatSecure/Classes/View Controllers/OTRMessagesViewController.h

```diff
@@ -52,6 +54,12 @@
 - (nullable OTRXMPPManager *)xmppManagerWithTransaction:(nonnull YapDatabaseReadTransaction *)transaction;
 - (nullable id <OTRMessageProtocol,JSQMessageData>)messageAtIndexPath:(nonnull NSIndexPath *)indexPath;
 
+- (BOOL)canSendMedia;
```

Comment:
This should be removed because the state should be reflected accurately. It shouldn't return true just because it's a group chat.